### DISCUSSION TITLE
Is it possible to collaborate across 2 devices?  Mirroring Lokl sites, content and docker containers?

### DISCUSSION THREAD

**Discussion by jordan-rich** [elementor/lokl#45] [Q&A]
Setting up Lokl on one computer (my desktop) is a breeze! I use it to push some sites up to a static host like Netlify. (also a piece of cake). 

On some occasions, say I travel out of town and I need to do a quick update to a site and II already have Lokl and docker already installed on my laptop. How can I "tap" into the same "instance" or container of the Wordpress sites and content to work on it Locally and push to the same Netlify host?

Basically it's collaboration I'm trying to do. 

I totally understand that this is not like a traditional host where things are online always. Lokl _is_ Local. (I get it).

But somehow is it possible to "sync" all files locally with something like  git? or Syncthing.. or maybe remote desktop? Hopefully this makes sense. 

Thanks in advance for any help. 

Btw thanks for such a phenomenal project!
